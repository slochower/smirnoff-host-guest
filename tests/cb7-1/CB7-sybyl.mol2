@<TRIPOS>MOLECULE
Cpptraj
  126   147     1     0     0
SMALL
No Charge or Current Charge


@<TRIPOS>ATOM
      1 N1          22.5060    20.7200    18.2390 N.am       1 CB7      -0.065801
      2 N2          23.2950    18.3410    18.6650 N.am       1 CB7      -0.065801
      3 N3          22.6460    20.3890    15.9970 N.am       1 CB7      -0.065801
      4 N4          23.2260    18.0210    16.3980 N.am       1 CB7      -0.065801
      5 N5          21.6180    19.8780    13.8290 N.am       1 CB7      -0.065801
      6 N6          22.0880    17.4730    14.2490 N.am       1 CB7      -0.065801
      7 N7          19.9820    19.2210    12.4430 N.am       1 CB7      -0.065801
      8 N8          20.4120    16.8510    12.8970 N.am       1 CB7      -0.065801
      9 N9          17.5500    18.5780    12.6260 N.am       1 CB7      -0.065801
     10 N10         17.9960    16.1290    12.7050 N.am       1 CB7      -0.065801
     11 N11         15.6130    18.2770    13.6240 N.am       1 CB7      -0.065801
     12 N12         15.9200    15.8060    13.5930 N.am       1 CB7      -0.065801
     13 N13         13.9620    18.0080    15.3630 N.am       1 CB7      -0.065801
     14 N14         14.2210    15.5690    15.3290 N.am       1 CB7      -0.065801
     15 N15         12.9510    17.9150    17.3700 N.am       1 CB7      -0.065801
     16 N16         13.4090    15.4570    17.4000 N.am       1 CB7      -0.065801
     17 N17         12.8070    18.1160    19.9020 N.am       1 CB7      -0.065801
     18 N18         13.4770    15.6820    19.8830 N.am       1 CB7      -0.065801
     19 N19         13.5870    18.4970    21.9390 N.am       1 CB7      -0.065801
     20 N20         14.4150    16.1380    21.8770 N.am       1 CB7      -0.065801
     21 N21         15.5520    19.2210    23.1920 N.am       1 CB7      -0.065801
     22 N22         16.3800    16.8570    23.3780 N.am       1 CB7      -0.065801
     23 N23         17.7040    19.8930    23.2370 N.am       1 CB7      -0.065801
     24 N24         18.4570    17.4910    23.3580 N.am       1 CB7      -0.065801
     25 N25         19.9120    20.3620    22.1760 N.am       1 CB7      -0.065801
     26 N26         20.7870    18.0350    22.5540 N.am       1 CB7      -0.065801
     27 N27         21.3770    20.6490    20.4860 N.am       1 CB7      -0.065801
     28 N28         22.3710    18.4170    20.9520 N.am       1 CB7      -0.065801
     29 C1          22.0430    21.1200    16.9890 C.2        1 CB7       0.583885
     30 C2          23.1370    17.3690    17.6410 C.2        1 CB7       0.583885
     31 C3          23.5610    19.7020    18.1070 C.3        1 CB7      -0.168175
     32 H1          24.5220    20.0610    18.4930 H          1 CB7       0.165028
     33 C4          23.6310    19.4070    16.6420 C.3        1 CB7      -0.168175
     34 H2          24.5690    19.6500    16.1290 H          1 CB7       0.165028
     35 C5          22.5640    20.7450    14.5040 C.3        1 CB7      -0.253967
     36 H3          22.2040    21.7790    14.4480 H          1 CB7       0.154667
     37 H4          23.5740    20.5900    14.1070 H          1 CB7       0.154667
     38 C6          23.2910    17.3720    15.1220 C.3        1 CB7      -0.253967
     39 H5          23.4590    16.2980    15.2580 H          1 CB7       0.154667
     40 H6          24.1530    17.7620    14.5690 H          1 CB7       0.154667
     41 C7          20.4160    20.2350    13.2810 C.2        1 CB7       0.583885
     42 C8          21.1530    16.4300    13.9290 C.2        1 CB7       0.583885
     43 C9          22.0360    18.5960    13.2660 C.3        1 CB7      -0.168175
     44 H7          23.0300    18.5680    12.8060 H          1 CB7       0.165028
     45 C10         20.8420    18.1280    12.2930 C.3        1 CB7      -0.168175
     46 H8          21.2480    17.9930    11.2840 H          1 CB7       0.165028
     47 C11         18.6450    19.2510    11.8690 C.3        1 CB7      -0.253967
     48 H9          18.7040    18.7180    10.9130 H          1 CB7       0.154667
     49 H10         18.4630    20.3200    11.7120 H          1 CB7       0.154667
     50 C12         19.3860    16.0400    12.2210 C.3        1 CB7      -0.253967
     51 H11         19.3950    16.2390    11.1440 H          1 CB7       0.154667
     52 H12         19.6820    14.9860    12.1800 H          1 CB7       0.154667
     53 C13         16.6250    19.2020    13.4220 C.2        1 CB7       0.583885
     54 C14         17.1940    15.2620    13.4370 C.2        1 CB7       0.583885
     55 C15         17.2180    17.2430    12.2260 C.3        1 CB7      -0.168175
     56 H13         17.1430    17.2350    11.1330 H          1 CB7       0.165028
     57 C16         15.7980    17.0650    12.8630 C.3        1 CB7      -0.168175
     58 H14         15.0330    16.8930    12.0970 H          1 CB7       0.165028
     59 C17         14.2610    18.5920    14.0720 C.3        1 CB7      -0.253967
     60 H15         13.5060    18.2850    13.3380 H          1 CB7       0.154667
     61 H16         14.1740    19.6840    14.1050 H          1 CB7       0.154667
     62 C18         14.6730    15.1250    14.0030 C.3        1 CB7      -0.253967
     63 H17         14.8770    14.0560    14.1280 H          1 CB7       0.154667
     64 H18         13.8540    15.2710    13.2890 H          1 CB7       0.154667
     65 C19         13.6300    18.7640    16.5360 C.2        1 CB7       0.583885
     66 C20         14.2470    14.8240    16.5070 C.2        1 CB7       0.583885
     67 C21         13.3270    16.6660    15.3530 C.3        1 CB7      -0.168175
     68 H19         12.5550    16.5150    14.5900 H          1 CB7       0.165028
     69 C22         12.7110    16.6100    16.8430 C.3        1 CB7      -0.168175
     70 H20         11.6350    16.4000    16.8200 H          1 CB7       0.165028
     71 C23         12.1780    18.3540    18.5820 C.3        1 CB7      -0.253967
     72 H21         12.3420    19.4340    18.4970 H          1 CB7       0.154667
     73 H22         11.1390    18.0060    18.6040 H          1 CB7       0.154667
     74 C24         12.9870    14.9420    18.6900 C.3        1 CB7      -0.253967
     75 H23         11.8940    14.9840    18.7560 H          1 CB7       0.154667
     76 H24         13.4110    13.9460    18.8610 H          1 CB7       0.154667
     77 C25         13.1960    19.0980    20.7990 C.2        1 CB7       0.583885
     78 C26         14.4780    15.2790    20.7470 C.2        1 CB7       0.583885
     79 C27         12.6670    16.8010    20.5190 C.3        1 CB7      -0.168175
     80 H25         11.6250    16.4630    20.5470 H          1 CB7       0.165028
     81 C28         13.2650    17.0760    21.8650 C.3        1 CB7      -0.168175
     82 H26         12.5870    16.8960    22.7070 H          1 CB7       0.165028
     83 C29         14.0770    19.2390    23.1400 C.3        1 CB7      -0.253967
     84 H27         13.7310    20.2760    23.0540 H          1 CB7       0.154667
     85 H28         13.6310    18.7450    24.0110 H          1 CB7       0.154667
     86 C30         15.2080    15.9180    23.1560 C.3        1 CB7      -0.253967
     87 H29         14.6120    16.0660    24.0640 H          1 CB7       0.154667
     88 H30         15.7490    14.9660    23.1260 H          1 CB7       0.154667
     89 C31         16.4580    20.2190    22.7490 C.2        1 CB7       0.583885
     90 C32         17.6520    16.4360    23.1140 C.2        1 CB7       0.583885
     91 C33         16.3020    18.1920    23.8960 C.3        1 CB7      -0.168175
     92 H31         15.8890    18.2940    24.9060 H          1 CB7       0.165028
     93 C34         17.7470    18.6210    23.9560 C.3        1 CB7      -0.168175
     94 H32         18.0380    18.8030    24.9970 H          1 CB7       0.165028
     95 C35         18.9390    20.6820    23.2600 C.3        1 CB7      -0.253967
     96 H33         18.5460    21.6910    23.0920 H          1 CB7       0.154667
     97 H34         19.4000    20.5120    24.2400 H          1 CB7       0.154667
     98 C36         19.9550    17.3390    23.5010 C.3        1 CB7      -0.253967
     99 H35         20.2360    16.2830    23.4290 H          1 CB7       0.154667
    100 H36         20.2020    17.6510    24.5230 H          1 CB7       0.154667
    101 C37         20.0970    20.9730    20.9790 C.2        1 CB7       0.583885
    102 C38         21.5700    17.4450    21.5880 C.2        1 CB7       0.583885
    103 C39         21.0500    19.5210    22.6200 C.3        1 CB7      -0.168175
    104 H37         21.4820    19.8910    23.5560 H          1 CB7       0.165028
    105 C40         22.0790    19.7770    21.4520 C.3        1 CB7      -0.168175
    106 H38         22.9590    20.3220    21.8110 H          1 CB7       0.165028
    107 C41         22.2000    21.4090    19.5290 C.3        1 CB7      -0.253967
    108 H39         21.7480    22.3570    19.2160 H          1 CB7       0.154667
    109 H40         23.1080    21.5890    20.1170 H          1 CB7       0.154667
    110 C42         23.5430    18.0590    20.1000 C.3        1 CB7      -0.253967
    111 H41         24.3780    18.6840    20.4380 H          1 CB7       0.154667
    112 H42         23.7970    17.0200    20.3360 H          1 CB7       0.154667
    113 O1          22.9770    16.1590    17.8520 O.2        1 CB7      -0.504502
    114 O2          21.0860    21.9040    16.6760 O.2        1 CB7      -0.504502
    115 O3          19.7960    21.2530    13.4470 O.2        1 CB7      -0.504502
    116 O4          20.9390    15.3180    14.5350 O.2        1 CB7      -0.504502
    117 O5          21.5690    16.2650    21.2830 O.2        1 CB7      -0.504502
    118 O6          16.6050    20.4040    13.6880 O.2        1 CB7      -0.504502
    119 O7          17.5890    14.2180    13.9110 O.2        1 CB7      -0.504502
    120 O8          13.8830    19.9750    16.7740 O.2        1 CB7      -0.504502
    121 O9          14.8000    13.7620    16.6810 O.2        1 CB7      -0.504502
    122 O10         13.3970    20.3110    20.5720 O.2        1 CB7      -0.504502
    123 O11         15.3170    14.3870    20.5630 O.2        1 CB7      -0.504502
    124 O12         16.2790    21.2860    22.2530 O.2        1 CB7      -0.504502
    125 O13         18.0650    15.3160    22.7860 O.2        1 CB7      -0.504502
    126 O14         19.3410    21.7270    20.4420 O.2        1 CB7      -0.504502
@<TRIPOS>BOND
     1   103   105 1   
     2   102   117 2   
     3   101   126 2   
     4    91    93 1   
     5    90   125 2   
     6    89   124 2   
     7    79    81 1   
     8    78   123 2   
     9    77   122 2   
    10    67    69 1   
    11    66   121 2   
    12    65   120 2   
    13    55    57 1   
    14    54   119 2   
    15    53   118 2   
    16    43    45 1   
    17    42   116 2   
    18    41   115 2   
    19    31    33 1   
    20    30   113 2   
    21    29   114 2   
    22    28   102 am  
    23    28   105 1   
    24    28   110 1   
    25    27   101 am  
    26    27   105 1   
    27    27   107 1   
    28    26    98 1   
    29    26   102 am  
    30    26   103 1   
    31    25    95 1   
    32    25   101 am  
    33    25   103 1   
    34    24    90 am  
    35    24    93 1   
    36    24    98 1   
    37    23    89 am  
    38    23    93 1   
    39    23    95 1   
    40    22    86 1   
    41    22    90 am  
    42    22    91 1   
    43    21    83 1   
    44    21    89 am  
    45    21    91 1   
    46    20    78 am  
    47    20    81 1   
    48    20    86 1   
    49    19    77 am  
    50    19    81 1   
    51    19    83 1   
    52    18    74 1   
    53    18    78 am  
    54    18    79 1   
    55    17    71 1   
    56    17    77 am  
    57    17    79 1   
    58    16    66 am  
    59    16    69 1   
    60    16    74 1   
    61    15    65 am  
    62    15    69 1   
    63    15    71 1   
    64    14    62 1   
    65    14    66 am  
    66    14    67 1   
    67    13    59 1   
    68    13    65 am  
    69    13    67 1   
    70    12    54 am  
    71    12    57 1   
    72    12    62 1   
    73    11    53 am  
    74    11    57 1   
    75    11    59 1   
    76    10    50 1   
    77    10    54 am  
    78    10    55 1   
    79     9    47 1   
    80     9    53 am  
    81     9    55 1   
    82     8    42 am  
    83     8    45 1   
    84     8    50 1   
    85     7    41 am  
    86     7    45 1   
    87     7    47 1   
    88     6    38 1   
    89     6    42 am  
    90     6    43 1   
    91     5    35 1   
    92     5    41 am  
    93     5    43 1   
    94     4    30 am  
    95     4    33 1   
    96     4    38 1   
    97     3    29 am  
    98     3    33 1   
    99     3    35 1   
   100     2    30 am  
   101     2    31 1   
   102     2   110 1   
   103     1    29 am  
   104     1    31 1   
   105     1   107 1   
   106   110   111 1   
   107   110   112 1   
   108   107   108 1   
   109   107   109 1   
   110   105   106 1   
   111   103   104 1   
   112    98    99 1   
   113    98   100 1   
   114    95    96 1   
   115    95    97 1   
   116    93    94 1   
   117    91    92 1   
   118    86    87 1   
   119    86    88 1   
   120    83    84 1   
   121    83    85 1   
   122    81    82 1   
   123    79    80 1   
   124    74    75 1   
   125    74    76 1   
   126    71    72 1   
   127    71    73 1   
   128    69    70 1   
   129    67    68 1   
   130    62    63 1   
   131    62    64 1   
   132    59    60 1   
   133    59    61 1   
   134    57    58 1   
   135    55    56 1   
   136    50    51 1   
   137    50    52 1   
   138    47    48 1   
   139    47    49 1   
   140    45    46 1   
   141    43    44 1   
   142    38    39 1   
   143    38    40 1   
   144    35    36 1   
   145    35    37 1   
   146    33    34 1   
   147    31    32 1   
@<TRIPOS>SUBSTRUCTURE
     1 CB7         1 TEMP              0 ****  ****    0 ROOT
