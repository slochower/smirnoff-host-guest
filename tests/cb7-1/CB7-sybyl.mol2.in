#!/usr/bin/env bash
source $AMBERHOME/amber.sh
    
        antechamber -i ./tests/cb7-1/CB7.mol2 -fi mol2 -o ./tests/cb7-1/CB7-sybyl.mol2 -fo mol2 -at sybyl -dr n
        