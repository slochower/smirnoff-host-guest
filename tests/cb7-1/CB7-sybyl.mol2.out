
Welcome to antechamber 17.3: molecular input file processor.


